packages/toast/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@lidofinance/toast",
+  "version": "0.0.0-semantic-release",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "license": "MIT",
+  "homepage": "https://github.com/lidofinance/ui",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lidofinance/ui.git",
+    "directory": "packages/toast"
+  },
+  "bugs": {
+    "url": "https://github.com/lidofinance/ui/issues"
+  },
+  "keywords": [
+    "lido",
+    "lidofinance",
+    "styled-components",
+    "component",
+    "react-component",
+    "react"
+  ],
+  "dependencies": {
+    "@lidofinance/accordion": "workspace:*",
+    "@lidofinance/button": "workspace:*",
```

Comment:
Can we move these two dependencies to devDependencies, since they are only used in stories?

Reply:
of course